### ISSUE TITLE
[A11y Compliance] SfButton - default color is not good enough for contrastivity

### ISSUE THREAD

**Issue by mayashavin** [DivanteLtd/storefront-ui#110]
Color contrast is not good enough (default button color should be slightly darker)
<img width="1445" alt="Screen Shot 2019-05-24 at 9 07 49" src="https://user-images.githubusercontent.com/6650139/58309879-0335a500-7e0e-11e9-8095-bacedd8c00a2.png">
